src/web/lib/components/App/index.js

```diff
@@ -1,20 +1,21 @@
-import React from "react";
+import React, { Fragment } from "react";
```

Comment:
Fragments are pretty new, but super handy...https://reactjs.org/docs/fragments.html#short-syntax
Basically let you return unwrapped components.